coralproject/talk
# New tenant utilities script for installing new tenants

## What does this PR do?

Adds a script under `utilities/tenant` that includes an installation script for new tenants on existing clusters that already have tenants.

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?

None

## Does this PR introduce any new environment variables or feature flags?

No

## If any indexes were added, were they added to `INDEXES.md`?

No

## How do I test this PR?

- perform a development build of coral using `sh scripts/build-development.sh`
- `cd` into `utilities/tenant`
- run `npm i`
- perform a tenant install via ` npm run start -- --mongoURI mongodb://localhost:27017 --mongoDBName coral createTenant`
- follow the prompts for a new tenant
- check your Mongo db to see the new tenant was created
- change the domain of the new tenant to localhost
- modify the existing coral tenant to not be localhost
- start up Coral
- see that the new tenant is now being used instead of the old one

## Where any tests migrated to React Testing Library?

None

## How do we deploy this PR?

No special considerations.
